model/src/main/proto/topic.proto

```diff
@@ -239,6 +239,55 @@ message TopicProgress {
 
   // Map from story ID to StoryProgress.
   map<string, StoryProgress> story_progress = 2;
+
+  // Timestamp to record last time the exploration was played in ms.
+  int64 last_played_timestamp = 4;
+}
+
+// A structure corresponding to the Promoted Stories. This structure is set up
+// to properly account for recently_played stories, for recommended stories
+// and for coming soon topics.
+message RecommendedActivityList {
+  oneof recommendation_type {
+    RecommendedStoryList recommended_story_list = 1;
+    ComingSoonTopicList coming_soon_topic_list = 2;
+  }
+}
+
+// Corresponds to the list of stories the player is currently playing across all topics and Recommended stories.
+message RecommendedStoryList {
+   // Ongoing stories from within the last 7 days.
+  repeated PromotedStory recently_played_story   = 1;
+
+  // Other ongoing stories from longer than 7 days ago.
+  repeated PromotedStory older_played_story = 2;
+
+  // Stories recommended to the user.
+  repeated PromotedStory suggest_story = 3;
```

Comment:
Should be: ``suggested_story``to keep consistent verb tense.

Reply:
Done.